products/angularjs.md

```diff
@@ -97,6 +97,6 @@ AngularJS was [deprecated](https://docs.angularjs.org/misc/version-support-statu
 December 31, 2021 after a [LTS period](https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c)
 on the final version 1.8.3 which was released April 7, 2022.
 
-Beginning January 1, 2022, commercial [extended long term support for AngularJS](https://www.herodevs.com/support/nes-angularjs)
-became available through the [HeroDevs](https://www.herodevs.com/) Never-Ending Support
-initiative (NES, formerly XLTS).
+Beginning January 1, 2022, commercial [extended long term support for AngularJS](https://www.herodevs.com/support/nes-angularjs) became available for 1.5.x and 1.8.x versions through a fork of the project maintained by the independent team at [HeroDevs](https://www.herodevs.com/) (formerly XLTS.dev).
```

Comment:
I don't agree with those changes, because:

- We aim to maintain a neutral and informative tone on endoflife.date, without promotional intent. For this reason the extended support description is always succinct, with a link if the user wants more information.
- angular versions can't be found on https://www.herodevs.com/support/nes-angularjs, so it was already removed in #3643.

Reply:
Thank you @marcwrobel for pointing that out, once we update the Website with the version I will refresh the PR